doc/whatsnew/2.14.rst

```diff
@@ -74,6 +74,9 @@ Other Changes
 
 * The concept of checker priority has been removed.
 
+* Add a new command line option ``--minimal-messages-config`` for ``pytest``, which disables all
+  irrelevant messages when running the functional tests.
+
```

Comment:
```suggestion
```
It's internal so I think it could be omitted from the changelog. This is something to take into account when we do #5728